netstandard/ref/System.Data.cs

```diff
@@ -1302,7 +1302,7 @@ public enum ParameterDirection
         Output = 2,
         ReturnValue = 6,
     }
-    public partial class PropertyCollection : System.Collections.Hashtable
+    public partial class PropertyCollection : System.Collections.Hashtable, System.ICloneable
```

Comment:
Why add it here when Hashtable is ICloneable and it other PRs we go exactly opposite way of removing inherited interfaces?

Reply:
This is a result of diffing from what is .NET Core. I'd be fine excluding this interface here but at the end of the day it doesn't really matter it will just add diff noise each time we diff against core.

Reply:
I've excluded `, System.ICloneable` for the iface list of `PropertyCollection`.